### PR TITLE
adding post about member's wall

### PR DESCRIPTION
**Pull request by julien51**
# Description

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
image

File: unlock-protocol.com/blog/member-wall.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+The Unlock team spent the week-end in Waterloo, Canada [hacking on Ethereum](https://unlock-protocol.com/blog/ethwaterloo-tickets/).
+
+I worked on building a "member wall", which is just a way to show the list of members for any given lock, as an image.
+
+# What does it really do?
+
+It's a very simple application which generates a large SVG image which lists all members of a key. For example, here is the list of [participants to the EthWaterloo hackathon](https://member-wall.unlock-protocol.com/api/members?locks=0xb0ad425ca5792dd4c4af9177c636e5b0e6c317bf&maxHeight=300), or the list of people who supported Evan Van Ness' [Week In Ethereum anniversary edition](https://member-wall.unlock-protocol.com/api/members?locks=0x79C91241eFf1F119CDf743730f6e6fB2aF7Fb279&maxHeight=300)!
+
+The great thing about using images, is that they can be embeded on almost any webpage. For example, below is the list of all members to our lock and that image is dynamically refreshed when new members are added!
```
Comment (cnasc) [inline unlock-protocol.com/blog/member-wall.md]:
```suggestion
The great thing about using images is that they can be embedded on almost any webpage. For example, below is the list of all members to our lock and that image is dynamically refreshed when new members are added!
```

File: unlock-protocol.com/blog/member-wall.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+<object data="https://member-wall.unlock-protocol.com/api/members?locks=0xB0114bbDCe17e0AF91b2Be32916a1e236cf6034F&maxHeight=300" type="image/svg+xml"/>
+
+Each of the member is clickable and will link to the corresponding page on [Etherscan](https://etherscan.io/)!
```
Comment (cnasc) [inline unlock-protocol.com/blog/member-wall.md]:
```suggestion
Each of the members is clickable and will link to the corresponding page on [Etherscan](https://etherscan.io/)!
```

File: unlock-protocol.com/blog/member-wall.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+![Members](https://member-wall.unlock-protocol.com/api/members?locks=0xB0114bbDCe17e0AF91b2Be32916a1e236cf6034F&maxHeight=30)
+
+Of couse, this is all [completely open source](https://github.com/unlock-protocol/member-wall), so you could easily run that on your own server, or maybe even host it on IPFS?
+
+# Embedding!
+
+One of the benfits of decoupling the membership from the platform itself means that this can actually be embedded on any website. For example, we added this list of members to [our Github page](https://github.com/unlock-protocol/unlock/), so that everyone who makes a donation is instantly shown on the README, as soon as the transaction gets mined!
+
+Where would you embed your member's wall?
```
Comment (cnasc) [inline unlock-protocol.com/blog/member-wall.md]:
```suggestion
Where would you embed your members wall?
```

**Pull request merged**